client/src/views/telemetry-view.tsx

```diff
@@ -40,16 +38,15 @@ export default function TelemetryView() {
 	useEffect(() => {
 		if (frequencySet) {
 			const interval = setInterval(async () => {
-				const data = await getTelemetryData();
-				console.log(data.header);
-				const { altitude, latitude, longitude, satellites, timeStamp } = data.header;
-				console.log(altitude, latitude, longitude, satellites, timeStamp);
-				setLongitude(longitude);
-				setLatitude(latitude);
-				setAltitude(altitude);
-				setSatelliteCount(satellites);
-				setTimeStamp(timeStamp);
-				setTelemetryData(data);
+				// console.log(data.header);
+				// const { altitude, latitude, longitude, satellites, timeStamp } = data.header;
+				// console.log(altitude, latitude, longitude, satellites, timeStamp);
+				// setLongitude(longitude);
+				// setLatitude(latitude);
+				// setAltitude(altitude);
+				// setSatelliteCount(satellites);
+				// setTimeStamp(timeStamp);
+				// setTelemetryData(data);
 			}, 1000);
 			return () => clearInterval(interval);
 		}
```

Comment:
kill this

Reply:
will do moma

Reply:
just realized I did this in a different branch